projects/packages/my-jetpack/src/class-rest-products.php

```diff
@@ -142,6 +163,16 @@ public static function get_products() {
 		return rest_ensure_response( $response, 200 );
 	}
 
+	/**
+	 * Site products endpoint.
+	 *
+	 * @return array of site products list.
+	 */
+	public static function get_products_by_ownership() {
+		$response = Products::get_products_by_ownership();
```

Comment:
This function requires either "unowned" or "owned" string argument. Most probably, what you want to achieve is:
```php
array(
  'unownedProducts' => Products::get_products_by_ownership( 'unowned' ),
  'ownedProducts'   => Products::get_products_by_ownership( 'owned' )
)
```